package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stripe/terminal-js",
-  "version": "0.8.0",
+  "version": "0.9.0",
```

Comment:
so… the publish to npm finished locally so we are actually on `0.9.0`, the push to master failed because the tree was dirty due to my local `.yarnrc` 😞